catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -545,35 +546,14 @@ protected ResourceInfo getResourceInfo(
 
     validateResourceInfoRequest(name, value);
 
+    if (value instanceof URI) {
```

Comment:
✏️ I think this check could go away to if we determine the URI from the queried metacard.